cli/list_strip_symbols.h

```diff
@@ -3,8 +3,9 @@
 #include "jl_exported_funcs.inc"
 #include "trampolines/common.h"
 #define XX(x) --strip-symbol=CNAME(x)
-JL_EXPORTED_FUNCS(XX)
+JL_RUNTIME_EXPORTED_FUNCS(XX)
 #ifdef _OS_WINDOWS_
-JL_EXPORTED_FUNCS_WIN(XX)
+JL_RUNTIME_EXPORTED_FUNCS_WIN(XX)
 #endif
+JL_CODEGEN_EXPORTED_FUNCS(XX)
 #undef XX
```

Comment:
I forget we did this. I think we can delete this now?

Reply:
@staticfloat

Reply:
Mmmm, yeah, we probably won't run into the same symbol conflicts anymore.  Let's try it!